thomasbrueggemann/paperless-desktop
# Adding support for images

## Describe the changes you made

I added support for images. This closes #31 

## Why were these changes necessary/desired?

Because it only supported PDFs so far.

## How can the changes be tested?

You have to upload an image. Click an image document, see that it loads nicely. Go back and click a PDF doc, see that they still load nicely.

## Did you already test the changes?

I tested with jpgs and pngs.
